person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly ensure that the value of rentals cannot only be read but also changed by adding it to ```attr_accessor``` as this would help in future projects.